packager/packager.cc

```diff
@@ -1176,102 +1162,170 @@ struct PackageInfoFinder {
     }
 };
 
-unique_ptr<PackageInfoImpl> rewritePackageSpec(const core::GlobalState &gs, ast::ParsedFile &package) {
+unique_ptr<PackageInfoImpl> definePackage(const core::GlobalState &gs, ast::ParsedFile &package) {
```

Comment:
This function is going to be the skeleton of the portion of this code that will move into namer.